### PR TITLE
Release v0.3.0 -- ESA CDSE cutover and timstamp fixes

### PR DESCRIPTION
**Pull request by jhkennedy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [0.2.5]
 
+### Update
+* Updated workflows to hyp3lib v2.0.2, which uses the new Copernicus Dataspace Ecosystem API got download orbit files.
+* Calls to `downloadSentinelOrbitFile` to specify the `esa_credentials` argument.
+
+### Added
+* `check_esa_credentials` function to `__main__.py` to check for the existence of Dataspace credentials before processing begins.
```
Comment (jhkennedy) [inline CHANGELOG.md:16]:
```suggestion
## [0.3.0]

### Added
* `check_esa_credentials` function to `__main__.py` to check for the existence of Dataspace credentials before processing begins.

### Changed
* Updated `hyp3lib` to  v2.0.2+, which uses the new Copernicus Dataspace Ecosystem API to download orbit files.
* Calls to `downloadSentinelOrbitFile` now specify the `esa_credentials` argument.
```

**Comment by cmarshak** [comment]
Questions:

1. Are the tests failing because of issues with downloading?
2. We have a big merge in #122 - should we wait to release those together or do them separately? Thinking that should be merged before turkey day? 🤞

**Comment by jhkennedy** [comment]
> 1. Are the tests failing because of issues with downloading?

Something threw a 504; I suspect it's transient and this round of checks to pass

> 2. We have a big merge in https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/pull/122 - should we wait to release those together or do them separately? Thinking that should be merged before turkey day? 🤞

I think we should get this out the door as soon as you're happy with these changes -- scihub orbits may shutdown at anytime so it'd be good to have already cut over. 

Always better to release smaller, incremental changes faster :-D

**Review by cmarshak — APPROVED** [review]
Looks great - thank you, @jhkennedy, @asjohnston-asf, and @forrestfwilliams.

**Pull request merged**